### DISCUSSION TITLE
Invalid player data

### DISCUSSION THREAD

**Discussion by LucanDRK** [TownyAdvanced/Towny#6929] [Q&A] [answered]
hello, I'm using your great plugin for creating a towny server, but if I quit and rejoin multiple times when the towny plugin is active, the server block every player that try to join saying ''invalid player data'', any help? (sorry for my english I'm Italian)

**Comment by DrunkBRIGO** [comment]
idk

**Comment by LlmDl** [comment, marked as answer]
Towny does not cause this problem you are seeing.
